src/routes/give-away/get.js

```diff
@@ -0,0 +1,53 @@
+import Joi from 'joi'
+import RestifyErrors from 'restify-errors'
+
+import models from '../../models'
+
+export default {
+
+  method: 'get',
+
+  // @TODO: How to filter?
+  path: '/giveaways?',
+
+  parameters: Joi.object().keys({
+    active: Joi.boolean(),
```

Comment:
This doesn't appear to be used?

Reply:
not yet, i want to filter out inactive giveaways. e.g., all of the editions have been claimed. then there's no point in returning it.